src/components/molecules/Cards/ItemCard.vue

```diff
@@ -3,20 +3,24 @@
   <div
     :data-id="item.id"
     class="handle flex justify-between items-center"
-    draggable="true"
+    :draggable="dropzone"
     @dragend="dropped"
+    @mousemove="selectedText"
   >
     <component
+      id="droptext"
       :is="'p'"
       :data-id="item.id"
       @focusout="update"
       contenteditable="true"
       class="prose prose-a:text-blue-600 break-all"
       v-html="htmlstring"
       ref="element"
+      draggable="false"
+      @mouseleave="selectedText"
     ></component>
     <div class="flex justify-end">
-      <HandIcon />
+      <HandIcon @click="dropzone = true" />
```

Comment:
Nothing to set it back to false

Reply:
When the user can select item text then I have set a false value for the dropzone variable
Please refer -> [https://i.imgur.com/lZZ1Bd9.png](url)